src/courseware/sequence/Sequence.jsx

```diff
@@ -27,27 +28,64 @@ function Sequence({
   bannerText,
   intl,
 }) {
+  const currentIndex = unitIds.indexOf(activeUnitId);
+
+  const logEvent = (eventName, widgetPlacement, targetUnitId) => {
+    const payload = {
+      current_tab: currentIndex + 1,
```

Comment:
```suggestion
    const payload = {
      // tabs are tracked with a 1-indexed position as opposed to a 0-index used throughout this mfe
      current_tab: currentIndex + 1,
```
May be worth commenting why we're adding one. I was hoping to keep our 1-indexed "position" out of the mfe but this seems unavoidable

Reply:
Good call, comment added above ^